math/Cargo.toml

```diff
@@ -11,6 +11,7 @@ license.workspace = true
 thiserror = { version = "1.0", optional = true }
 serde = { version = "1.0", features = ["derive"], optional = true }
 serde_json = { version = "1.0", optional = true }
+proptest = "1.1.0"
```

Comment:
Please, make this dependency optional and use a feature flag. We don't want to force the dependency for users who aren't interested in testing with arbitrary numbers.

Reply:
Resolved!